README.md

```diff
@@ -42,7 +42,16 @@ represent the tiles.
 
 ## A1.1
 
-«replace this with your answer»
+S-> A
+
+S-> A S A
+
+A-> r
+
+A-> g
+
+A-> b
+
```

Comment:
You're on the right lines, but in "A S A" the two A's do not have to be the same (if they did, then every digit in `number = digit+` would also have to be the same).

You need something like

~~~
       S -> r | g | b
       S -> r  S  r
       S -> g  S  g
       S -> b  S  b
~~~

# -3